dependent-service-downtime-and-ps-adaptor-behavior.md

```diff
@@ -0,0 +1,55 @@
+## Service Failures and Expected Behavior
+
+This guide outlines the Behavior Of The Requesting Adaptor when various dependent services are down. 
+The Requesting Adaptor works in conjunction with several services including the GP2GP Translator, MHS Adaptor, and others. 
+The guide explains the effects of outages and how the system behaves in each case.
+
+### System Overview
+The Requesting Adaptor initiates a conversation for transferring patient records, relying on several interconnected services:
+- GP2GP Translator: Sends patient records to the Requesting Adaptor via a queue.
+- Facade Application: Works with the Requesting Adaptor to manage transfer requests.
+- MHS Adaptor: Manages inbound and outbound queues, which are responsible for transferring patient records to the incumbent system.
+- Other Dependencies: Include Requesting Adaptor database, MHS inbound and outbound services and File storage.
+
+### Service Outage Scenarios And Behavior Of The Adaptor In Certain Scenarios
+1. GP2GP Translator is Down
+   - Scenario: The GP2GP Translator is not operational and cannot send patient records.
+   - Expected Behavior: No messages are sent to the incumbent system while the translator is down. Once the GP2GP Translator recovers, 
+     the transfer process resumes, and messages are processed as normal.
+   - Recovery: Automatic upon Translator recovery; transfer resumes without manual intervention.
+2. Message Broker is Down
+   - Scenario: The message broker responsible for transferring data between the GP2GP Translator and Requesting Adaptor is down.
+   - Expected Behavior: The initial request is never sent to the incumbent system because the queue is unavailable.
+                        The facade responds with 500 "Internal Server Error". The transfer wasn't sent to the sending adapter.
+                        The transfer can be requested again once the message broker is up and running.
+   - Recovery: After the queue is restored, the request can be processed normally, but any messages queued during downtime are not sent.
```

Comment:
This only talks about the failure in the context of starting a new conversation. It doesn't cover what happens to transfers which are currently in progress.